modules/protect.php

```diff
@@ -142,52 +142,76 @@ public function maybe_display_security_warning() {
+			wp_send_json_error( new WP_Error( 'insufficient_permissions' ) );
+		}
+
+		update_site_option( 'jetpack_dismissed_protect_multisite_banner', true );
 
-		wp_enqueue_style( 'jetpack', plugins_url( "css/jetpack-banners{$min}.css", JETPACK__PLUGIN_FILE ), false, JETPACK__VERSION );
-		$wp_styles->add_data( 'jetpack', 'rtl', true );
+		wp_send_json_success();
 	}
 
+	/**
+	 * Displays a warning about Jetpack Protect's network activation requirement.
+	 * Attaches some custom JS to Core's `is-dismissible` UI to save the dismissed state.
+	 */
 	public function admin_jetpack_manage_notice() {
-
-		$dismissed = get_site_option( 'jetpack_dismissed_protect_multisite_banner' );
-
-		if ( $dismissed ) {
-			return;
-		}
-
-		$referer     = '&_wp_http_referer=' . add_query_arg( '_wp_http_referer', null );
-		$opt_out_url = wp_nonce_url( Jetpack::admin_url( 'jetpack-notice=jetpack-protect-multisite-opt-out' . $referer ), 'jetpack_protect_multisite_banner_opt_out' );
-
 		?>
-		<div id="message" class="updated jetpack-message jp-banner is-opt-in protect-error"
-		     style="display:block !important;">
-			<a class="jp-banner__dismiss" href="<?php echo esc_url( $opt_out_url ); ?>"
-			   title="<?php esc_attr_e( 'Dismiss this notice.', 'jetpack' ); ?>"></a>
-
-			<div class="jp-banner__content">
-				<h2><?php esc_html_e( 'Protect cannot keep your site secure.', 'jetpack' ); ?></h2>
-
-				<p><?php printf( __( 'Thanks for activating Protect! To start protecting your site, please network activate Jetpack on your Multisite installation and activate Protect on your primary site. Due to the way logins are handled on WordPress Multisite, Jetpack must be network-enabled in order for Protect to work properly. <a href="%s" target="_blank">Learn More</a>', 'jetpack' ), 'http://jetpack.com/support/multisite-protect' ); ?></p>
-			</div>
-			<div class="jp-banner__action-container is-opt-in">
-				<a href="<?php echo esc_url( network_admin_url( 'plugins.php' ) ); ?>" class="jp-banner__button"
-				   id="wpcom-connect"><?php _e( 'View Network Admin', 'jetpack' ); ?></a>
-			</div>
+		<div class="jetpack-protect-warning notice notice-warning is-dismissible" data-dismiss-nonce="<?php echo esc_attr( wp_create_nonce( 'jetpack_protect_multisite_banner_opt_out' ) ); ?>">
+			<h2><?php esc_html_e( 'Jetpack Brute Force Attack Prevention cannot keep your site secure', 'jetpack' ); ?></h2>
+
+			<p><?php esc_html_e( "Thanks for activating Jetpack's brute force attack prevention feature! To start protecting your whole WordPress Multisite Network, please network activate the Jetpack plugin. Due to the way logins are handled on WordPress Multisite Networks, Jetpack must be network activated in order for the brute force attack prevention feature to work properly.", 'jetpack' ); ?></p>
+
+			<p>
+				<a class="button-primary" href="<?php echo esc_url( network_admin_url( 'plugins.php' ) ); ?>">
+					<?php _e( 'View Network Admin', 'jetpack' ); ?>
```

Comment:
should we "esc" translations, sometimes we do and sometimes.
```suggestion
					<?php esc_html_e( 'View Network Admin', 'jetpack' ); ?>
```